docs/blueprints/EP002.rst

```diff
@@ -1,13 +1,6 @@
-##################
-OpenFlow Handshake
-##################
-
-
-Status
-######
-
-Accepted
-
+:EP: BP002
+:Title: OpenFlow Handshake
+:Status: accepted 
```

Comment:
As this is already an effort to standardize these headers, I think it would be nice to stick to one standard. :) E.g.: capitalized for Status.
```suggestion
:Status: Accepted 
```

Reply:
Ok, I updated the others types of status too.